lib/config-generator.js

```diff
@@ -270,6 +270,17 @@ export default [\n${exportContent}];`;
                 choices: ["npm", "yarn", "pnpm", "bun"]
             })).packageManager;
 
+            // removing `--force` when:
+            // 1. yarn v2+ -- it does not support it.
```

Comment:
We don't know if the user is using yarn v1 or yarn v2+, but I've tested yarn v1 is also working fine without `--force`.